keyboards/tanuki/tanuki.c

```diff
@@ -0,0 +1,17 @@
+#include "tanuki.h"
+#include "rgblight.h"
+#include "config.h"
+//#include "led.h"
+#include "rgblight.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	//LED_TYPE led[RGBLED_NUM];
+	//setrgb(255, 255, 255, led);
+	//rgblight_show_solid_color(255,255,255);
+	rgblight_enable();
```

Comment:
I'm not sure that this should be in the kb function. It may be best to put this into the default keymap, using the `matrix_init_user` function instead.